traffic_portal/app/src/traffic_portal_properties.json

```diff
@@ -306,71 +306,71 @@
           },
           "rangeRequestHandling": {
             "title": "Range Request Handling",
-            "desc": "How to treat range requests.<br><br><a href=\"http://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_ops/using.html#delivery-service\" target=\"_blank\">See more...</a>"
+            "desc": "How to treat range requests.<br><br><ul><li>0: Do not cache (ranges requested from files that are already cached due to a non range request will be a HIT)</li><li>1: Use the <a href=\"https://docs.trafficserver.apache.org/en/latest/admin-guide/plugins/background_fetch.en.html\" target=\"_blank\">background_fetch plugin.</a></li><li>2: Use the cache_range_requests plugin.</li></ul>"
           },
           "regexRemap": {
             "title": "Regex remap expression",
-            "desc": "Regex Remap rule to apply to this delivery service at the Edge tier.<br><br><a href=\"http://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_ops/using.html#delivery-service\" target=\"_blank\">See more...</a>"
+            "desc": "Allows remapping of incoming requests URL using regex pattern matching to search/replace text.<br><br><a href=\"https://docs.trafficserver.apache.org/en/latest/admin-guide/plugins/regex_remap.en.html\" target=\"_blank\">See ATS documentation on regex remap</a><b>Note:<b> You will not be able to save a Regex Remap Expression if you have Query String Handling set to drop query strings at the edge. <a href=\"http://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_ops/using.html#regex-remap-expression\" target=\"_blank\">See Regex Remap Expression</a>"
           },
           "regionalGeoBlocking": {
             "title": "Regional Geoblocking",
-            "desc": "Description required"
+            "desc": "(For HTTP routed delivery services only) Define regional geo-blocking rules for delivery services in a JSON format and set it to True/False. <a href=\"http://traffic-control-cdn.readthedocs.io/en/latest/admin/quick_howto/regionalgeo.html#configure-regional-geo-blocking-rgb\" target=\"_blank\">See Regional Geo-blocking</a>"
           },
           "remapText": {
             "title": "Raw remap text",
             "desc": "For HTTP and DNS delivery services, this will get added to the end of the remap line on the cache verbatim. For ANY_MAP delivery services this is the remap line."
           },
           "routingName": {
             "title": "Routing Name",
-            "desc": "The routing name to use for the delivery FQDN, i.e. routing-name.deliveryservice.cdn-domain. It must be a valid hostname without periods."
+            "desc": "The routing name to use for the delivery FQDN, i.e. routing-name.deliveryservice.cdn-domain. It must be a valid hostname without periods. Typical values would be ccr, edge and cdn."
```

Comment:
There are no typical values I don't think. It can be whatever you want it to be.